requirements.txt

```diff
@@ -215,8 +215,8 @@ reportlab==3.4.0 \
 requests==2.21.0 \
     --hash=sha256:502a824f31acdacb3a35b6690b5fbf0bc41d63a24a45c4004352b0242707598e \
     --hash=sha256:7bf2a778576d825600030a110f3c0e3e8edc51dfaafe1c146e39a2027784957b
-SQLAlchemy==1.0.12 \
-    --hash=sha256:6679e20eae780b67ba136a4a76f83bb264debaac2542beefe02069d0206518d1
+SQLAlchemy==1.3.3 \
```

Comment:
SQLAlchemy 1.3.4 was just released so you may want to update this again

Reply:
Done and tested